src/main/java/mat/server/FeatureFlagServiceImpl.java

```diff
@@ -13,7 +13,7 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-@Service
+@Service("featureFlagService")
```

Comment:
why do we need a name? just curious. it's usually not required

Reply:
we don't need this. was experimenting with something else. this service is registered twice with spring container. Need refactor.